src/soilbiogeochem/SoilBiogeochemDecompCascadeCNMod.F90

```diff
@@ -39,6 +39,18 @@ module SoilBiogeochemDecompCascadeCNMod
   integer, private            :: i_soil3   = -9         ! SOM third pool
   integer, private            :: i_soil4   = -9         ! SOM fourth pool
 
+  real(r8), private :: cwd_fcel
```

Comment:
we don't need this in CNMod, since the CN model is being deprecated.

Reply:
True, but I think that the case would have stopped in the build phase, so I made the changes anyway.